### PR TITLE
Improve layout

### PR DESCRIPTION
**Pull request by lentil32**
Use code block to print result
Use flexbox layout
Use wide layout

31b3592b02295beec565ff1400e351ae6af30ab337db93cfd9e90721b2f8f7b6

### CODE REVIEW

**Comment by lentil32** [comment]
[Add error control](https://github.com/rla1wo23/Kubernetes-yaml-detector/pull/2/commits/dbaebb4ca1f0e4e1e5792822607a60d88c6ecd3d)

image

**Comment by rla1wo23** [comment]
바로반영했네 굿 LGTM

File: yaml-detector.py [hunk 1]
```diff
@@ -2,26 +2,53 @@
...
+    st.header(TEXT["HEADER"])
+    st.text(TEXT["NOTICE"])
 
-if q or btn:
-    result = fix_code(q)
-    print(f"Result: {result}")
+    col1, col2, col3 = st.columns([3, 1, 3])
 
-st.text_area("To", value=result)
+    with col1:
```
Comment (rla1wo23) [inline yaml-detector.py:26]: 에러핸들링에 gpt를 쓸까 생각도 해봄 에러뜨면 case gpt로 분기하는식으로

**Review by rla1wo23 — APPROVED** [review]
LGTM

**Pull request merged**